### PR TITLE
Fix definition of ChangesCountMap

### PR DESCRIPTION
**Pull request by arskama**
ChangesCountMap is a map with source as key
and state changes count as value.

Fixes: #226

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -375,7 +375,10 @@ <h3>
       a <dfn>[[\PenaltyDuration]]</dfn> integer set as part of the [=reset observation window=] steps.
     </li>
     <li>
-      a <dfn>[[\ChangesCountMap]]</dfn> integer set as part of the [=reset observation window=] steps.
+      a <dfn>[[\ChangesCountMap]]</dfn> [=ordered map=], [=map/keyed=] on a {{PressureSource}},
+      representing the [=source type=] to which the last state change belongs.
```
Comment (anssiko) [inline index.html:379]: Would this proposed alternative wording mean the same:

`representing the [=source type=] that triggered transition to the [=current pressure state=].`

If that makes sense, then on L385 how about this wording:

`representing the [=source type=] of the last {{PressureRecord}}.`

(Disclaimer: I read this PR without additional context just trying make those lines read more like English, so this may not be what you try to specify.)

**Review by anssiko — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -375,7 +375,10 @@ <h3>
...
+      The [=ordered map=]'s [=map/value=] is an integer representing the count of state changes in the
```
Comment (kenchris) [inline index.html:380]: maybe "amount of"

**Review by kenchris — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (arskama) [reply index.html:379]: ok, Thanks I ll make the change!

File: index.html [hunk 3]
```diff
@@ -375,11 +375,14 @@ <h3>
       a <dfn>[[\PenaltyDuration]]</dfn> integer set as part of the [=reset observation window=] steps.
     </li>
     <li>
-      a <dfn>[[\ChangesCountMap]]</dfn> integer set as part of the [=reset observation window=] steps.
+      a <dfn>[[\ChangesCountMap]]</dfn> [=ordered map=], [=map/keyed=] on a {{PressureSource}},
+      representing the [=source type=] that triggered transition to the [=current pressure state=].
+      The [=ordered map=]'s [=map/value=] is an integer representing the amount of state changes in the
```
Comment (anssiko) [inline index.html:380]: s/amount of state changes/number of state changes

**Review by anssiko — APPROVED** [review]
Approved with a grammar improvement suggestion.
